cves/kernel/CVE-2018-13406.yml

```diff
@@ -473,5 +510,5 @@ nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
```

Comment:
Can't comment on unchanged code, so I'm putting this here:

Line 506: Make sure you confirm the CWE listed.

Reply:
CWE number is already present